src/sentry/snuba/metrics.py

```diff
@@ -447,16 +447,40 @@ class SnubaField:
         "operations": _AVAILABLE_OPERATIONS["metrics_sets"],
         "tags": _SESSION_TAGS,
     },
+    "transaction.duration": {
+        "type": "distribution",
+        "operations": _AVAILABLE_OPERATIONS["metrics_distributions"],
+        "tags": {
+            **_MEASUREMENT_TAGS,
+            "transaction.status": [
+                # Subset of possible states:
+                # https://develop.sentry.dev/sdk/event-payloads/transaction/
+                "ok",
+                "cancelled",
+                "aborted",
+            ],
+        },
+    },
 }
 
 _METRICS.update(
     {
-        f"measurements.{web_vital}": {
+        f"measurements.{measurement}": {
             "type": "distribution",
             "operations": _AVAILABLE_OPERATIONS["metrics_distributions"],
             "tags": _MEASUREMENT_TAGS,
         }
-        for web_vital in ("lcp", "fcp", "fid", "cls")
+        for measurement in (
+            "lcp",
+            "fcp",
+            "fid",
+            "cls",
+            "frames_total",
+            "frames_slow",
+            "frames_frozen",
```

Comment:
Looks like there's also `frames_slow_rate` and `frames_frozen_rate` (slow and total divided).
The current performance product uses those two. Can we have an alias like that too?

Reply:
I've refactored to use an existing list of metric names, which includes both: https://github.com/getsentry/sentry/blob/050e66ff0beb3c7d2b92824ca385ba55dfc48b65/src/sentry/relay/config.py#L368-L369